src/ewatercycle/_forcings/caravan.py

```diff
@@ -0,0 +1,240 @@
+import xarray as xr
+import numpy as np
+import pandas as pd
+from cartopy.io import shapereader
+import fiona
+
+from pathlib import Path
+from typing import Type
+
+import requests
+from zipp import zipfile
+
+from ewatercycle.base.forcing import DefaultForcing, LumpedUserForcing
+from ewatercycle.util import get_time
+
+COMMON_URL = "ca13056c-c347-4a27-b320-930c2a4dd207"
+OPENDAP_URL = f"https://opendap.4tu.nl/thredds/dodsC/data2/djht/{COMMON_URL}/1/"
+SHAPEFILE_URL = f"https://data.4tu.nl/file/{COMMON_URL}/bbe94526-cf1a-4b96-8155-244f20094719"
+
+PROPERTY_VARS = ['timezone',
```

Comment:
The code formatter doesn't like this.

You can run it with "pre-commit":
- `pip install pre-commit`
- `pre-commit run -a`

Usually it only looks at staged files, but with the `-a` flag you can run it on all files. This should also fix the CI.

Reply:
Still getting issues with CI:
```
nbqa-isort...............................................................Failed
..
ModuleNotFoundError: No module named 'pkg_resources'
```
 and 
```
nbqa-flake8..............................................................Failed
...
AttributeError: 'EntryPoints' object has no attribute 'get'
```